### ISSUE TITLE
Netflix does not start

### ISSUE THREAD

**Issue by NickJH** [CastagnaIT/plugin.video.netflix#901]
#### Your Environment
- Netflix add-on version: 1.10.0
- Operating system version/name: OSMC (Buster), Kodi 18.8 
- Device model:  RPi 3B+

Used Operating system:
* [ ] Android
* [ ] iOS
* [ ] Linux
* [ ] OSX
* [*] Raspberry-Pi
* [ ] Windows

### Describe the bug
I installed v1.0.1 of your repo using a zip install, then installed Netflix 1.10.0 and all the other suggested packages. When I go to start the addon, the circle goes round for a while then stops and nothing happens. Initially I tried to resolve it by installing python3-crypto. That didn't work so I removed it and installed python3-pycryptodome. That also failed. The logs are with python3-pycryptodome installed.
[kodi.log](https://paste.kodi.tv/icelavuhow.kodi)

#### Expected behavior
Netflix should start

#### Actual behavior
The circle goes round for a bit then stops.

**Comment by Quandm1195** [comment]
I can confirm the same issue, the loading icon got stuck even though the contents are loaded. Therefore you can't see anything. It was working fine but the OSMC update in 10/2020 (a couple of days ago) might have interfered. 
Workaround is to change the OSMC skin to the default one. Then I can view the content.

**Comment by NickJH** [comment]
In my case I was using a Confluence skin. I have switched back to OSMC default, rebooted and there is no change. It looks like it is loading then does nothing.

**Comment by CastagnaIT** [comment]
>                                          ImportError: No module named Crypto.Random

is missing a crypto package
why you do not want to install pycryptodomex?

> I tried to resolve it by installing python3-crypto

Kodi 18 works only with python 2 not with python 3...

**Comment by NickJH** [comment]
I didn't say I didn't want to install pycryptodomex. I tried googling around to see what package supplied the Crypto.Random package. During the upgrade I noticed python3 packages being installed so I assumed I'd need python3-pycryptodome. I am not a python ot Kodr expert.

The question is then, why isn't pycryptodomex as one of the packages pulled in as a dependency of the installation? It looks like the package needed is `python-pycryptodome`.

I have now installed it and Netflix worked once I rebooted it.
